default/config.json

```diff
@@ -26,6 +26,7 @@
         "font-size": "16px"
     },
     "highlight-modes": "",
+    "lang": "",
```

Comment:
Could you please explain why you added this line?

Reply:
Is it redundant?
I think the configuration file should contain the necessary settings.
If it is an error, just ignore it please, cause I am a Freshman and it is hard for me to read JavaScript.

Reply:
Well, I think this is out of the scope of this PR, but it sounds ok. I'll merge it.